src/Chunk.cpp

```diff
@@ -1715,10 +1715,12 @@ void cChunk::SetAlwaysTicked(bool a_AlwaysTicked)
 	if (a_AlwaysTicked)
 	{
 		m_AlwaysTicked += 1;
+		Stay(a_AlwaysTicked);
```

Comment:
`m_AlwaysTicked` is always smaller or equal to `m_StayCount`. A chunk may have chunkStay without being always ticked.